packages/lesswrong/components/ea-forum/users/EAUsersMetaInfo.tsx

```diff
@@ -22,6 +23,9 @@ const styles = (theme: ThemeType): JssStyles => ({
       color: theme.palette.grey[600],
       "&:hover": {
         color: theme.palette.grey[600],
+      },
+      "&:visited": {
+        color: theme.palette.grey[600],
```

Comment:
This is also to make sure the user's location doesn't turn purple (it should stay grey)